website_img_bg_style/views/snippets.xml

```diff
@@ -0,0 +1,165 @@
+                        <a href="#">Position</a>
+                        <ul class="dropdown-menu">
+                            <li class="dropdown-submenu">
+                                <a href="#">Vertical</a>
+                                <ul class="dropdown-menu">
+                                    <li data-select_class="bg-top">
+                                        <a>Top</a>
+                                    </li>
+                                    <li data-select_class="bg-center-h">
+                                        <a>Center</a>
+                                    </li>
+                                    <li data-select_class="bg-bottom">
+                                        <a>Bottom</a>
+                                    </li>
+                                    <li data-select_class="">
+                                        <a>None</a>
+                                    </li>
+                                </ul>
+                            </li>
+                            <li class="dropdown-submenu">
+                                <a href="#">Horizontal</a>
+                                <ul class="dropdown-menu">
+                                    <li data-select_class="bg-left">
+                                        <a>Left</a>
+                                    </li>
+                                    <li data-select_class="bg-center-v">
+                                        <a>Center</a>
+                                    </li>
+                                    <li data-select_class="bg-right">
+                                        <a>Right</a>
+                                    </li>
+                                    <li data-select_class="">
+                                        <a>None</a>
+                                    </li>
+                                </ul>
+                            </li>
+                        </ul>
+                    </li>
+
+                    <li class="dropdown-submenu">
+                        <a href="#">Repeat</a>
+                        <ul class="dropdown-menu">
+                            <li class="dropdown-submenu">
+                                <a href="#">Vertical</a>
+                                <ul class="dropdown-menu">
+                                    <li data-select_class="bg-repeat-v">
+                                        <a>Clipped</a>
+                                    </li>
+                                    <li data-select_class="bg-space-v">
+                                        <a>Spaced</a>
+                                    </li>
+                                    <li data-select_class="bg-round-v">
+                                        <a>Rounded</a>
+                                    </li>
+                                    <li data-select_class="bg-no-repeat-v">
+                                        <a>None</a>
```

Comment:
Here I miss the `<li data-select_class="">` option, although you can call it *Auto* this time, to avoid confusions of having 2 *None* options.